pkg/scheduler/framework/plugins/noderesources/most_allocated.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package noderesources
+
+import (
+	"context"
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/kubernetes/pkg/scheduler/algorithm/priorities"
+	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/migration"
+	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
+)
+
+// NodeResourcesMostAllocated is a score plugin that favors nodes with high allocation based on requested resources.
+type NodeResourcesMostAllocated struct {
+	handle framework.FrameworkHandle
+}
+
+var _ = framework.ScorePlugin(&NodeResourcesMostAllocated{})
+
+// NodeResourcesMostAllocatedName is the name of the plugin used in the plugin registry and configurations.
+const NodeResourcesMostAllocatedName = "NodeResourcesMostAllocated"
```

Comment:
We could cut the noderesources from the constant and struct, `noderesoures.MostAllocatedName` could be better compare to `noderesources.NdoeResourcesMostAllocatedName`